src/dtclient/dynatrace_client.go

```diff
@@ -13,6 +13,8 @@ import (
 	"github.com/pkg/errors"
 )
 
+var ErrHostNotFound = errors.New("host not found")
```

Comment:
isn't this incorrect? shouldn't we create a `type` and use `errors.As` to check it, isn't that safer/nicer ?

I complained about it in @StefanHauth 's PR [here](https://github.com/Dynatrace/dynatrace-operator/pull/2129#discussion_r1329957567), sooooo I wanna know, if its me who does not know something 😅

Reply:
fixed

Reply:
fixed in 6bd00cc6

Reply:
Its probably safer, but checking for a special error later on isn't nicer because it always requires creating an extra var. For simple things, I'm also more in favor if this style. While it's ofc not impossible, I'm not sure how likely it is, that some overwrites this var.